airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/java/io/airbyte/cdk/integrations/source/jdbc/test/JdbcSourceAcceptanceTest.java

```diff
@@ -202,9 +205,13 @@ protected void maybeSetShorterConnectionTimeout(final JsonNode config) {
 
   @AfterEach
   public void tearDown() {
+    customCleanUp();
+    dropSchemas();
```

Comment:
the call to dropSchemas will significantly impact the speed of the CI loop for certified connectors. Please protect this call with an overridable shouldDropSchemaAfterTest or something similar